lib/ttnt/test_selector.rb

```diff
@@ -29,6 +32,8 @@ def select_tests
       diff = @base_obj.diff(@target_obj)
       diff.each_patch do |patch|
         file = patch.delta.old_file[:path]
+        tests << file and next if test_file?(file)
```

Comment:
> Also feel free to ask me regarding your FIXME and TODO comments ; I do not understand them. :p

I'm sorry for unclear comments :sweat: I will check them out and ask you soon (maybe via another issue)